.dockerignore

```diff
@@ -1,10 +1,13 @@
 #Ignore Kv files
 config/Kv*
 
-#Ignore build directories
-workloads/build*
-tcs/core/common/build*
-tcs/core/tcs_trusted_worker_manager/enclave/build*
-tcs/core/common/python/build*
-common/build*
+#Ignore documentation files
```

Comment:
You can use *.md  and docs/\* to exclude all documentation files.

Reply:
Done